maths/is_square_free.py

```diff
@@ -0,0 +1,42 @@
+'''
+References: wikipedia:square free number
+'''
+from typing import List
+
+def is_square_free(factors:List[int]) -> bool:
+    '''
+    This functions takes a list of prime factors as input.
+    returns True if the factors are square free.
+
+    >>> is_square_free([1,1,2,3,4])
+    False
+
+    These are wrong but should return some value
+    it simply checks for repition in the numb
+    >>> is_square_free([1,3,4,'sd', 0.0])

```

Comment:
`<space> between list items`

Reply:
When I use `from prime_factors import prime_factors` the build fails. Why is that?

Reply:
Put a . (dot) as in __from .prime_factors import prime_factors__.